src/main/resources/application.properties

```diff
@@ -1,3 +1,8 @@
 # suppress inspection "UnusedProperty" for whole file
 application.namespace=file-transfer-service
-spring.servlet.multipart.max-file-size=${MAX_FILE_SIZE:300MB}
\ No newline at end of file
+spring.servlet.multipart.max-file-size=${MAX_FILE_SIZE:300MB}
+######### AWS S3 Credentials #########
+aws.secretAccessKey=${AWS_SECRET_ACCESS_KEY}
+aws.accessKeyId=${AWS_ACCESS_KEY_ID}
+aws.s3Path=${IMAGE_CLOUD_S3_PATH}
```

Comment:
I left a comment about the env var name on [the docker chs development pull request](https://github.com/companieshouse/docker-chs-development/pull/618), for completeness I'll mention here too that I think this anvironment variable would be better named as `S3_PATH`, `BUCKET_NAME`, or `FILE_UPLOAF_BUCKET` or something like that.

Reply:
done